### ISSUE TITLE
Use up-to-date glyphicons icon set

### ISSUE THREAD

**Issue by lolilolicon** [lordi/tickmate#65]
The newest [glyphicons free icon set](https://glyphicons.com/files/glyphicons_free.zip) contains 800 basic icons and 70 social icons, why not use it?

Why I want this: I was looking for a good while for the icon for "sleeping", but found none in the app. Later I found on the glyphicons site the "bed" and "bed-alt" icons, which will suit my use case but unfortunately aren't currently shipped by tickmate.

**Comment by lordi** [comment]
Thanks for the hint, I'll include the new icons soon.

**Comment by lolilolicon** [comment]
I also happen to notice `glyphicons_334_candle_white` is used as icon for `Meditation`; you can now make it use `glyphicons-623-meditation` instead :)

**Comment by lordi** [comment]
Theres a problem that glyphicons changed the filenames, image numbers and image sizes in this new iconset. I'll have to write a script that can detect matching old and new icons, and rename the new icons. Because the icon name is stored in the database for old users, and I don't want them to wake up with no icons :-)

**Comment by lordi** [comment]
There are also the github octions that could be used in addition to the new glyphicons: https://octicons.github.com/

**Comment by lolilolicon** [comment]
Yeah, isn't backward compatibility nasty? Imagine doing this again and again in the future! Maybe change the approach a bit, say, by making a local copy of the icon, whose filename reflects the task ID, every time the user selects one for a task? As a bonus, this approach can potentially allow using an external image for an icon, too... Wait, maybe you should consider this.

Now what if tickmate doesn't ship any icon at all, but allows the user to use whatever image as an icon? In the absence of a user-supplied icon, tickmate displays either the first character of the task name or perhaps the star icon.

Something along those lines. The advantages include smaller app size and less maintenance burden.

And I forgot to say, thank you for tickmate. It's been helpful to me :)

**Comment by lordi** [comment]
Another (more lightweight) way would be to use emoji icons: https://github.com/rockerhieu/emojicon/

**Comment by lolilolicon** [comment]
Yes, emoji should work pretty well! It's also easier to navigate as the icons are organized in groups rather than presented in a single large list.

For me personally, the first character of the task title, rendered in color, should be sufficient to distinguish tasks, although I don't know whether it's technically trivial to implement.

IMO either way should improve usability and maintainability over the current design. It's up to you :)

**Comment by freddii** [comment]
try font awesome:
http://fontawesome.io/cheatsheet/
https://github.com/encharm/Font-Awesome-SVG-PNG/tree/master/black/svg
